src/Ifelse.java

```diff
@@ -0,0 +1,199 @@
+        } else {
+            System.out.println("Het getal "+ number1 + " is even.");
+        }
+    }
+
+    // Opdracht 2
+    private static void opdracht2(float cijfer) {
+        if (cijfer >= 8.0f) {
+            System.out.println(cijfer + " A");
+        } else if (cijfer >= 7.7f) {
+            System.out.println(cijfer +  "A-");
+        } else if (cijfer >= 7.4f) {
+            System.out.println(cijfer + " B+");
+        } else if (cijfer >= 7.0f) {
+            System.out.println(cijfer + " B");
+        } else if (cijfer >= 6.7f) {
+            System.out.println(cijfer + " B-");
+        } else if (cijfer >= 6.4f) {
+            System.out.println(cijfer + " C+");
+        } else if (cijfer >= 6.0f) {
+            System.out.println(cijfer + " C");
+        } else if (cijfer >= 5.5f) {
+            System.out.println(cijfer + " C-");
+        } else {
+            System.out.println(cijfer + " F");
+        }
+    }
+
+    //Opdracht 3
+    private static void opdracht3(int count1, int count2) {
+        if (count1 > count2) {
+            System.out.println("De invoer van het tweede getal " + count1 + " groter getal dan de eerste invoer met getal " + count2);
+        } else if (count2 > count1) {
+            System.out.println("De invoer van het tweede getal " + count2 + " groter getal dan de eerste invoer met getal " + count1);
+        } else {
+            System.out.println("eerste invoer is gelijk" + count1 + "  met de tweede invoer " + count2 + "! ");
+        }
+    }
+
+    //Opdracht 4
+    private static void opdracht4(int number2) {
+        if (number2 > 0) {
+            System.out.println("Dit nummer " + number2 + " is positief");
+        } else if (number2 < 0) {
+            System.out.println("Dit nummer = " + number2 + " is negative");
+        } else {
+            System.out.println("Dit nummer = " + number2 + " is zero");
+        }
+    }
+
+    //Opdracht 5
+    private static void opdracht5(int leeftijd) {
+        if (leeftijd <= 12) {
+            System.out.println("Met deze leeftijd " + leeftijd + " ben je een kind");
+        } else if (leeftijd <= 18) {
+            System.out.println("Met deze leeftijd " + leeftijd + " ben je een puber");
+        } else if (leeftijd <= 69) {
+            System.out.println("Met deze leeftijd " + leeftijd + " ben je een volwassene");
+        } else {
+            System.out.println("Met deze leeftijd " + leeftijd + " ben je een senior");
+        }
+    }
+
+    //opdracht 6
+    private static void opdracht6(String text) {
+        if (text.length() <= 20) {
+            System.out.println("korter dan 10 tekens " + text);
+        } else if (text.length() <= 40) {
+            System.out.println("Tussen 10 en 20 tekens " + text);
+        } else {
+            System.out.println("langer dan 20 tekens " + text);
+        }
+    }
+
+    //opdracht 7
+    private static void opdracht7(int date) {
```

Comment:
Deze opdracht heb je niet helemaal correct uitgewerkt. Er zit wat meer nuance in dan "elke 4 jaar"